packages/rum-core/src/common/url.js

```diff
@@ -82,12 +91,12 @@ class Url {
        * Default values for all keys from location if url is relative
        */
       this[key] =
-        this[key] || (relative && instruction[2] ? location[key] || '' : '')
+        this[key] || (relative && instruction[3] ? location[key] || '' : '')
```

Comment:
I think these kind of changes are might be hard to follow, if we ever add another rule again, but maybe we can refactor at another time.

Reply:
I will add more documentation to the code to make it easier to follow.